### PR TITLE
alertHistoryIndex and findingIndexPattern in data sources

### PR DESCRIPTION
**Pull request by petardz**
Signed-off-by: Petar Dzepina <petar.dzepina@vroom.com>

### Description
Added alertHistoryIndex and findingIndexPattern to DataSources
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt [hunk 1]
```diff
@@ -22,6 +22,9 @@ data class DataSources(
      *  If index is pre-existing, mapping is updated. */
     val alertsIndex: String = ".opendistro-alerting-alerts", // AlertIndices.ALERT_INDEX
 
+    /** Configures a custom index to store historic alerts for a monitor.*/
+    val alertsHistoryIndex: String,
```
Comment (eirsep) [inline src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt:26]: let's add the default value used in alerting plugin

**Comment by eirsep** [comment]
plz address CI failure

File: src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt (hunk 1 above)
Comment (petardz) [reply src/main/kotlin/org/opensearch/commons/alerting/model/DataSources.kt]: done

**Review by eirsep — APPROVED** [review]

**Comment by eirsep** [comment]
plz raise PR to main branch. we will backport to 2.x

**Comment by eirsep** [comment]
https://github.com/opensearch-project/common-utils/pull/290
covered this changes

**Pull request closed without merging**